src/project-generator.ts

```diff
@@ -67,6 +71,9 @@ export class ProjectGenerator {
     public async run(codeSnippet: string, appName: string): Promise<number> {
         try {
             await this.generateRepo();
+            // Delay is introduced to make sure that the git API creates
+            // everything we need before doing other API requests
+            await this.delay(5000);
```

Comment:
Introduce timeout constant and use the same constant everywhere or is there a real reason for 5000ms vs 4000ms?

Reply:
Used same constant now and even reduced it a bit.